src/test/java/com/imsweb/x12/reader/X12ReaderTest.java

```diff
@@ -46,12 +48,26 @@ public void testConstructors() throws IOException {
         assertEquals(fromFileUtf8.getLoops().get(0).toString(), fromReaderUtf8.getLoops().get(0).toString());
     }
 
+    /**
+     * Here we will test that you can go from x12, make changes, then serialize the x12 once again.
+     */
+    @Test
+    public void testSerialize() throws IOException {
+        URL url = this.getClass().getResource("/837_5010/x12_valid.txt");
+
+        X12Reader fromFileUtf8 = new X12Reader(FileType.ANSI837_5010_X222, new File(url.getFile()), StandardCharsets.UTF_8);
+
+        fromFileUtf8.getSeparators().setLineBreak(LineBreak.LF);
+        String expected = IOUtils.toString(this.getClass().getResourceAsStream("/837_5010/x12_valid.txt"),
+            StandardCharsets.UTF_8).trim();
+        Assert.assertEquals(expected, fromFileUtf8.toX12String().trim());
+    }
+
```

Comment:
It would be good to add a unit test for a claim that has a Loop 2000C too. x12_complex.txt is an example of such a claim.

Reply:
i did this, and it reproduces the issue you discussed above. when i have it passing will check it in.